src/objects/media/media.stories.mdx

```diff
@@ -93,6 +94,44 @@ The order of `o-media__object` and `o-media__content` will not affect this patte
   </Story>
 </Canvas>
 
+## Alignment
+
+By default, content will appear middle-aligned with the object. If the content exceeds the object's height, they will appear top-aligned. You can change the alignment so the content and object remain top aligned using the modifier `o-media--alignStart`.
```

Comment:
It isn't very clear to me _why_ we would do this? 🤔

Reply:
When I updated the Case Study Listing prototype I noticed some of the media objects and content were top-aligned and some were middle-aligned. In a group like this, it looks off when the alignment is different.

![Screen Shot 2021-11-12 at 1 18 14 PM](https://user-images.githubusercontent.com/42841342/141537289-fd64fa0c-2178-4cd3-8574-4abb303e67bb.png)

Reply:
Ah, interesting. I wonder if that's better solved on the media object or on the deck by changing `align-items` or `align-content` via a modifier?

Reply:
Hmm, that's a good question. I don't have a strong opinion on whether we add a modifier to media or the deck. What do you think?

Reply:
I think it makes more sense on `o-deck` for a couple of reasons:

- The `o-media` object isn't really causing this alignment issue on its own: It's because the Deck is causing it to stretch to fill the column.
- I think `o-deck--align-start` is easier to understand than `o-media--align-start`, because to most people I think they'd look at `o-media` and think "isn't that already aligned to the start?" Whereas if you were to have a deck of cards I think it would be easier to see the difference.
- It's a little nicer to put the modifier on one container instead of every child element.